openqa-label-known-issues

```diff
@@ -175,7 +171,37 @@ investigate_issue() {
 }
 
 label_issue() {
-    local testurl="${1:?"Need 'testurl'"}"
+    local arg="${1:-}"
+    if [[ -z "$arg" ]]; then
+        echo "Usage: $0 <job_id> or https://<host>/tests/<job_id>" >&2
+        exit 1
```

Comment:
How about:
```suggestion
    local arg="${1:?"Usage: $0 <job_id> or https://<host>/tests/<job_id>"}"
```

Reply:
I'm not sure about this:

```
$ ./openqa-label-known-issues
./openqa-label-known-issues: line 174: 1: Usage: ./openqa-label-known-issues <job_id> or https://<host>/tests/<job_id>
$ ./openqa-label-known-issues
Usage: ./openqa-label-known-issues <job_id> or https://<host>/tests/<job_id>
```

the first run is with your suggestion and the second is with the current code.

As user I would prefer the second one because if I get the first one I would think about an unhandled error.